pyscf_ipu/experimental/special.py

```diff
@@ -77,23 +77,20 @@ def binom_lookup(x: IntN, y: IntN, nmax: int = LMAX) -> IntN:
 binom = binom_lookup
 
 
-def gammanu(nu: IntN, t: FloatN, epsilon: float = 1e-10) -> FloatN:
+def gammanu(nu: IntN, t: FloatN, num_terms: int = 128) -> FloatN:
```

Comment:
Maybe leave both in, called e.g. `gammanu_series` and `gammnu_gamma`, and switch between them as you do with binom.  Then the PR can check timings with a clean switch within the PR?